### DISCUSSION TITLE
迅雷的这个广告，可以定位，但是手机中并不触发怎么办？

### DISCUSSION THREAD

**Discussion by dearmb** [gkd-kit/gkd#797] [Q&A]
[Uploading snapshot-1732949721459.zip…]()
迅雷的这个广告，可以定位，但是手机中并不触发怎么办？
不会关闭，GKD记录中也没有

**Comment by dearmb** [comment]
好几种都不触发规则了

**Reply by lisonge** [reply]
zip 文件上传失败，GitHub 上看不到你说的快照

不触发需要设置 action 为 clickCenter 再不触发开启高级设置中的 强制点击

**Reply by lisonge** [reply]
https://gkd.li/api/interfaces/RawRuleProps#action

**Reply by dearmb** [reply]
我不会，我觉得是升级的问题，我备份了我的规则，重新降级安装到了1.9.1，app图标也好使了，刚刚说的迅雷新加的规则也好使了
